All EIPIP Meetings/Meeting 012.md

```diff
@@ -1,10 +1,10 @@
 # 1. EIP Improvement Process Call #12 Notes

```

Comment:
Remove 1. from above. It can be simply 
#  EIP Improvement Process Call #12 Notes